qfieldcloud_sdk/sdk.py

```diff
@@ -298,6 +473,19 @@ def upload_file(
         show_progress: bool,
         job_id: str = "",
     ) -> requests.Response:
+        """Upload a single file to a QFieldCloud project.
```

Comment:
```suggestion
        """Upload a single file to a project.
```

No need to specify "QFieldCloud Project", there are no other projects in this context.